### PR TITLE
Hotfix/2.6.11

### PR DESCRIPTION
**Pull request by chrisseto**

### CODE REVIEW

File: share/sources/org.biorxiv/source.yaml [hunk 1]
```diff
@@ -22,6 +22,15 @@ configs:
       namespaces:
           'http://purl.org/rss/1.0/': null
           'http://purl.org/dc/elements/1.1/': dc
+- base_url: http://biorxiv.org/search/
```
Comment (chrisseto) [inline share/sources/org.biorxiv/source.yaml]: Other should be disabled.

File: share/sources/org.biorxiv/source.yaml [hunk 2]
```diff
@@ -22,6 +22,15 @@ configs:
...
+  disabled: false
+  earliest_date: 2013-11-01
+  harvester: org.biorxiv.html
+  harvester_kwargs: {}
+  label: org.biorxiv.html
+  rate_limit_allowance: 1
+  rate_limit_period: 10
```
Comment (chrisseto) [inline share/sources/org.biorxiv/source.yaml]: I just realized that I had changed this back. I couldn't stand wait that long locally.

**Comment by laurenbarker** [comment]
Travis build is failing. I think `django-debug-toolbar` needs to be added to `dev-requirements.txt`

**Comment by chrisseto** [comment]
@laurenbarker the failures are coming ES failures. release/2.7.0 contains updated travis settings.

**Comment by laurenbarker** [comment]
Well that's what they are now because you commented out the debug_toolbar stuff ☝️

**Comment by laurenbarker** [comment]
Functionality looks good! I would use cursor pagination for `normalizeddata` and `rawdata` endpoints as well. Also an example on the `share_api` [docs page](http://share-research.readthedocs.io/en/latest/share_api.html).

**Pull request merged**